R/get_stats.R

```diff
@@ -180,3 +183,84 @@ get_wb <- function(year = "all",
 
   return(out)
 }
+
+#' @rdname get_stats
+#' @param aggregate character: Aggregate name. See `get_aux("countries")` for available options.
+#' @export
+get_agg <- function(year = "all",
+                   povline = NULL,
+                   version = NULL,
+                   ppp_version = NULL,
+                   release_version = NULL,
+                   aggregate = NULL,
+                   api_version = "v1",
+                   format = c("rds", "json", "csv"),
+                   simplify = TRUE,
+                   server = NULL) {
+
+  # Match args
+  api_version <- match.arg(api_version)
+  format <- match.arg(format)
+
+  # Extract varibale name that don't have "_code" and "_name" suffixes from countries auxiliary table
+  df <- get_aux("countries")
```

Comment:
This is not the right aux file. You need to use one called `country_list` or something like that. Thanks